api/DataTransfer.json

```diff
@@ -448,11 +448,12 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true,
+              "version_added": "3.5",
```

Comment:
`mozClearDataAt ` remains [in Gecko source](https://github.com/mozilla/gecko-dev/blob/e8d5a5ad644903b001c076279703a7a8f8a811ea/dom/webidl/DataTransfer.webidl#L101) but I guess `ChromeOnly` was added for Firefox 71.